.github/workflows/rust.yml

```diff
@@ -71,6 +71,16 @@ jobs:
           # we run tests using BitVec<u64,_> which doesn't.
           args: --all-features --target wasm32-unknown-unknown
 
+  diff-readme-files:
+    name: Check that READMEs are identical
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v3
+
+      - name: Diff READMEs
+        run: diff -q README.md scale-encode/README.md
```

Comment:
dq: why do we need two readme's?

Reply:
The one in `/scale-encode` is the oen that is shown on crates.io for the crate, and the one at root is shown on github! This was the laziest way I could think of to make sure that they were kept in sync :)

Reply:
you could just link to root README in Cargo.toml but fine :)

Reply:
Yep, probably a symlink would work here to avoid some extra CI minutes :D

Reply:
That _might_ not work because if the readme is outside of the crate it won't be uploaded with it, but I'd have to try it!

Reply:
Git does handle symlinks ok (as long as the FS supports it) though by the sounds of it (eg https://stackoverflow.com/questions/954560/how-does-git-handle-symbolic-links), so we could just make one readme be a symlink to the other, but I'm not too bothered either way given the CI check :)